.github/workflows/ci.yml

```diff
@@ -152,6 +152,7 @@ jobs:
         run: |
           git clone https://github.com/koute/bytehound
           cd bytehound
+          git checkout bcae7caa3b01bb60ec6d4159ac8b9af28a7c927a
```

Comment:
Add a brief comment explaining why we use this version?

Reply:
Done.